packages/gluestick/src/types.js

```diff
@@ -220,6 +220,8 @@ export type Hooks = {
   postGetCurrentRoute?: Hook;
   postRender?: Hook;
   error?: Hook;
+  webpackClientConfig?: Hook;
+  webpackServerConfig?: Hook;
```

Comment:
I think since we have webpack hooks in separated file those types should be also separated.